src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,32 @@
+type Props = {
+  errorType: string | null;
+  setErrorType: (errorType: string | null) => void;
+};
+
+export const ErrorNotification: React.FC<Props> = ({
+  setErrorType,
+  errorType,
+}) => {
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={
+        errorType !== null
+          ? 'notification is-danger is-light has-text-weight-normal '
+          : 'notification is-danger is-light has-text-weight-normal hidden'
+      }
```

Comment:
Use the classnames library for add classes with conditions. Also, now you are duplicating many of the same classes in the condition, you need to move them outside the condition